client/packages/system/src/Name/apiModern/hooks/useContacts.ts

```diff
@@ -0,0 +1,23 @@
+import { useQuery } from '@openmsupply-client/common';
+import { useContactGraphQL } from '../useContactsGraphQL';
+import { CONTACTS } from './keys';
+
+export const useContacts = (nameId: string) => {
+  const { contactApi, storeId } = useContactGraphQL();
+
+  const queryFn = async () => {
+    const result = await contactApi.contacts({ nameId, storeId });
+    if (result.contacts.__typename === 'ContactConnector') {
+      return result.contacts;
+    }
+  };
+
+  const { data, isLoading, isError } = useQuery({
+    queryKey: [CONTACTS],
+    queryFn,
+  });
+
+  return {
+    query: { data: data?.nodes ?? [], isLoading, isError },
+  };
```

Comment:
We're not going to be doing any mutations for this so we can just do 

`return { data: data?.nodes ?? [], isLoading, isError }`